### DISCUSSION TITLE
IIS on Nano?

### DISCUSSION THREAD

**Discussion by CynanX** [dotnet/dotnet-docker#5049] [Q&A]
Hey.

Currently using the Windows Core images with IIS and the image size is very large. I was wondering if the would could be reduced by using the nano images and installing IIS on top?

I saw an article that suggested IIS was now available on nano (https://learn.microsoft.com/en-us/iis/get-started/whats-new-in-iis-10/introducing-iis-on-nano-server) but I've had trouble getting this working and I think support might have been dropped in a later version of nano?

Is Nano + IIS possible? Would it reduce the image size or then be the same as Core and not worth it?

Thank you all.

**Comment by lbussell** [comment]
Hi @CynanX, you are correct - that article was written before NanoServer became container-only. Unfortunately, there isn't an easy way to install IIS on NanoServer today. The Windows container team is aware of the request to use IIS in NanoServer containers, but we can't make any promises yet. You'll have to stick with Server Core for now.
